cmw-dto/src/main/java/com/liberologico/cloudesire/cmw/model/dto/EnvironmentDTO.java

```diff
@@ -1564,6 +1566,13 @@ public boolean isForbidden()
         @ApiModelProperty( "How many subscriptions to allow per-user for a product" )
         private SubscriptionsPerProduct subscriptionsPerProduct;
 
+        @ApiModelProperty( "URL for the CSP API connector" )
+        private Map<CspProductType, String> cspApiEndpoints;
+
+        private String cspAzureProductId = "MS-AZR-0146P";
+
+        private String cspAzureParameterCode = "DNS_PREFIX";
```

Comment:
The code to use when autocreating the ConfigurationParameter for Azure CSP products